readme.txt

```diff
@@ -231,6 +231,7 @@ Remember to always make a backup of your database and files before updating!
 
 = [6.0.15] TBD =
 
+* Fix - Fix Fatal error: `Uncaught TypeError: array_search(): Argument #2 ($haystack) must be of type array, string given in /nas/content/live/website/wp-content/plugins/the-events-calendar/src/Tribe/Main.php:3535`. [TEC-4780]
```

Comment:
We should avoid making a repeat of the word `Fix` and should describe a little more to non-developers the error.

```suggestion
* Fix - Prevent administration navigation fatal error with `TypeError: array_search()`. [TEC-4780]
```